ham/ham.py

```diff
@@ -0,0 +1,404 @@
+# -*- coding: utf-8 -*-
+"""Ham: Monitor and test your OpenStack.
+
+usage: ham <command> [<args> ...] [-h|--help]
+           [-v|--version] [-vv|-s|--silent]
+
+Options:
+  -h --help      Show this help message.
+  -v --version   Show version number.
+  -vv            Verbose mode.
+  -s --silent    Quiet mode.
+
+Commands:
+  up             Get resources and install the docker registry.
+  os             Run kolla and install OpenStack.
+  init           Initialise OpenStack with the bare necessities.
+  bench          Run rally on this OpenStack.
+  ssh-tunnel     Print configuration for port forwarding with horizon.
+  info           Show information of the actual deployment.
+  deploy         Shortcut for ham up, then ham os and ham config.
+
+See 'ham <command> --help' for more information on a specific command.
+"""
+from . import __version__
```

Comment:
see my comment above, and let's stick with those relative imports for the moment.